src/main/kotlin/racingcar/view/InputView.kt

```diff
@@ -0,0 +1,18 @@
+package racingcar.view
+
+import racingcar.view.input.Input
+
+class InputView(private val input: Input) {
+    var carCount = 0
+        private set
+    var tryCount = 0
+        private set
+
+    fun start() {
+        println("자동차 대수는 몇 대인가요?")
+        this.carCount = input.readCar()
+
+        println("시도할 횟수는 몇 회인가요?")
+        this.tryCount = input.readTry()
+    }
```

Comment:
초기화 과정에 자원을 많이 사용하거나 객체를 사용할 때마다 꼭 초기화하지 않아도 되는 프로퍼티에 대해 지연 초기화(lazy initialization) 패턴을 사용할 수 있습니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/delegated-properties.html#lazy-properties

```suggestion
    val carCount: Int by lazy { askCarCount() }
    val tryCount: Int by lazy { askTryCount() }

    private fun askCarCount(): Int {
        println("자동차 대수는 몇 대인가요?")
        return input.readCar()
    }

    private fun askTryCount(): Int {
        println("시도할 횟수는 몇 회인가요?")
        return input.readTry()
    }
```

Reply:
오..! lazy initialization 을 지원하는군요! 감사합니다 :)